### DISCUSSION TITLE
Showcase:  The book "Learn JavaScript: Beginners Edition" was authored using GitBook

### DISCUSSION THREAD

**Discussion by sumn2u** [GitbookIO/community#160] [Show and Tell]
The content of the book ["Learn JavaScript: Beginners Edition"](https://leanpub.com/learnjavascriptbeginnersguide/) was authored using GitBook and is now publicly available at https://js201.gitbook.io/js-101/.

Expressing gratitude to GitBook 🙏.

**Comment by aikoii** [comment]
This is great! Thank you for sharing 😃.

**Comment by addisonschultz** [comment]
@sumn2u that is awesome! We would be more than happy to showcase it in our library if you submit it to us at https://www.gitbook.com/gitbook-library!
